src/main/java/com/google/devtools/build/bfg/JavaSourceFileParser.java

```diff
@@ -48,7 +62,28 @@
 
   private final Set<String> unresolvedClassNames;
 
-  private final ImmutableGraph<String> classDependencyGraph;
+  /**
+   * e.g., "com.Foo" --> "org.junit.Test".
+   *
+   * <p>(u, v) in G iff 'u' mentioned 'v' in code.
+   */
+  private final ImmutableGraph<String> classToClass;
```

Comment:
nit: Can you make this classToClassGraph or classToClassDepGraph?

Seeing as how the others are maps, it makes sense to incorporate this into the name.

Reply:
> I don't think this matter?

Yeah. Sounds good, it was a minor nit. :)